iOS/RCTOrientation/Orientation.h

```diff
@@ -12,5 +12,7 @@
 
 @interface Orientation : NSObject <RCTBridgeModule>
 + (void)setOrientation: (UIInterfaceOrientationMask)orientation;
++ (void)setOrientation: (UIInterfaceOrientationMask)orientation;
++ (void)updateInterfaceOrientation: (UIInterfaceOrientation)orientation withOrientationMask: (UIInterfaceOrientationMask) orientationMask;
```

Comment:
I thought about having only the `orientation` parameter and mapping it to an `orientationMask`. However, I realized there isn't always a clear mapping. In the code before any of my changes, sometimes `UIInterfaceOrientationMaskLandscape` was used with `UIInterfaceOrientationLandscapeLeft`, and sometimes `UIInterfaceOrientationMaskLandscapeLeft` was used with `UIInterfaceOrientationLandscapeLeft`. I figured it be easier to always pass in both the `orientation` and `orientationMask`. The `orientation` will get used on iOS 15 and below while the `orientationMask` will get used on iOS 16 and higher.